docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -134,25 +138,134 @@ PUT _ml/datafeeds/datafeed-farequote
 ----------------------------------
 // TEST[skip:setup:farequote_job]
 
-<1> The aggregations have names that match the fields that they operate on. The 
+<1> The aggregations have names that match the fields that they operate on. The
 `max` aggregation is named `time` and its field also needs to be `time`.
-<2> The `term` aggregation is named `airline` and its field is also named 
+<2> The `term` aggregation is named `airline` and its field is also named
 `airline`.
-<3> The `avg` aggregation is named `responsetime` and its field is also named 
+<3> The `avg` aggregation is named `responsetime` and its field is also named
 `responsetime`.
 
-Your {dfeed} can contain multiple aggregations, but only the ones with names 
-that match values in the job configuration are fed to the job.
+TIP: If you are utilizing a `term` aggregation to gather influencer or partition
+detector field information, consider using a `composite` aggregation. It performs
+better than a `date_histogram` with a nested `term` aggregation and also includes
+all the values of the field instead of the top values per bucket.
+
+[discrete]
+[[aggs-using-composite]]
+=== Using composite aggregations in {anomaly-jobs}
+
+experimental::[]
+
+For `composite` aggregation support, there must be exactly one `date_histogram` value
+source. That value source must not be sorted in descending order. Additional
+`composite` aggregation value sources are allowed, such as `terms`.
+
+NOTE: `composite` aggregations may not be as performant as a scrolling or
+`date_histogram` oriented {dfeed}. `composite` aggregations are optimized
+for queries that are either `match_all` or `range` filters. Other types of
+queries may cause the `composite` aggregation to be ineffecient.
+
+Here is an example utilizing a `composite` aggregation instead of a
```

Comment:
Not mandatory, but feels like an "uses" is simpler to understand and translate:
```suggestion
Here is an example that uses a `composite` aggregation instead of a
```